File: app/src/main/java/com/example/sensoresapp/MainActivity.java
package com.example.sensoresapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageButton;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ImageView btnCamara = (ImageView) findViewById(R.id.btnCamera);

        btnCamara.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent next = new Intent(MainActivity.this, CamaraActivity.class);
                startActivity(next);
                finish();
            }
        });
        ImageView btnvideo = (ImageView) findViewById(R.id.btnvideo);

        btnvideo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent next = new Intent(MainActivity.this, VideoActivity.class);
                startActivity(next);
                finish();
            }
        });

    }
}

File: app/src/main/java/com/example/appsem17/MainActivity.kt
package com.example.appsem17

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import androidx.cardview.widget.CardView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        
        val btnFor = findViewById<CardView>(R.id.forCiclo)
        val btnIf = findViewById<CardView>(R.id.ifCondicion)
        val btnDoWhile = findViewById<CardView>(R.id.doWhileCiclo)
        val btnWhile = findViewById<CardView>(R.id.whileCiclo)
        //val btnSwitch = findViewById<CardView>(R.id.switchCase)

        btnIf.setOnClickListener {
            val intent = Intent(this, IfActivity::class.java)
            startActivity(intent)
        }

        btnFor.setOnClickListener {
            val intent = Intent(this, ForActivity::class.java)
            startActivity(intent)
        }

        btnDoWhile.setOnClickListener {
        val intent = Intent(this, DoWhileActivity::class.java)
            startActivity(intent)
        }

        btnWhile.setOnClickListener {
        val intent = Intent(this, WhileActivity::class.java)
            startActivity(intent)
        }

        /*btnSwitch.setOnClickListener {
        val intent = Intent(this, SwitchActivity::class.java)
            startActivity(intent)
        }*/

    }
}